tests/DI/EventDispatcherExtensionTest.php

```diff
@@ -39,6 +39,8 @@ public function testLoadSubscribers()
 		$containerBuilder->addDefinition('eventSubscriber')
 			->setClass(SomeEventSubscriber::class);
 
+		$containerBuilder->prepareClassList();
```

Comment:
I didn't think it was necessary. The second test in this file already had prepareClassList in itself (without any note).